src/services/blockchain/ethereum/ethereum-blockchain-service.ts

```diff
@@ -115,20 +126,50 @@ function handleTimeoutError(transactionTimeoutSecs: number): void {
 function make(config: Config): EthereumBlockchainService {
   const ethereum = config.blockchain.connectors.ethereum
   const { host, port, url } = ethereum.rpc
+  let network
+  try {
+    network = Network.from(ethereum.network)
```

Comment:
hmm... this is likely to be a breaking change as now the network option is required in the config even when using an infura url (which implies the network and so network wasn't previously required).

That's fine since we're the only ones running a CAS, just something to look out for when we go to deploy.  We should double check we have the network set so things don't break on the upgrade

Reply:
Edit: I see it's not actually required as if its not set we fall back to the non-static behavior.  But that could cause our infura endpoint to be pummeled with chainid calls and blow up our bill (this happened in the past) so we'll still want to be really careful here and make sure have the network flag set on all 3 envs, and monitor our infura account as this gets deployed to various envs.